pom.xml

```diff
@@ -182,6 +182,11 @@ THE SOFTWARE.
       <version>1.12</version>
       <scope>provided</scope>
     </dependency>
+    <dependency>
+      <groupId>javax.websocket</groupId>
+      <artifactId>javax.websocket-api</artifactId>
+      <version>1.1</version>
```

Comment:
Might want to add `<scope>provided</scope>` since this is supposed to be bundled with the websocket implementation?